### PR TITLE
zh-translation: /about/contribute/creating-and-editing-pages/index.md

### PR DESCRIPTION
**Pull request by kylesliu**
Ref: https://github.com/servicemesher/istio-official-translation/issues/1640

### CODE REVIEW

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 1]
```diff
@@ -700,3 +758,7 @@ $ make lint
 {{< text bash >}}
 $ make INTERNAL_ONLY=True lint
 {{< /text >}}
+
+## 使用 GitHub{#using-github}
```
Comment (tanjunchen) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]: using-github
->
using-Github

Comment (kylesliu) [reply content/zh/about/contribute/creating-and-editing-pages/index.md]: https://istio.io/about/contribute/creating-and-editing-pages/#using-github
看了一下英文貌似是小写

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 2]
```diff
@@ -700,3 +758,7 @@ $ make lint
 {{< text bash >}}
 $ make INTERNAL_ONLY=True lint
 {{< /text >}}
+
+## 使用 GitHub{#using-Github}
```
Comment (gauliang) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
## 使用 GitHub{#using-GitHub}
```
试一下应该可以了

Comment (kylesliu) [reply content/zh/about/contribute/creating-and-editing-pages/index.md]: 已修改，请review

**Comment by tanjunchen** [comment]
@gauliang  can this merged?

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 3]
```diff
@@ -612,6 +612,64 @@ which looks like:
 
 - **没有嵌套的标签集**。不要尝试，这太可怕了。
 
+## 导航和表情{#banners-and-stickers}
+
+为了宣传宣传即将发生的事件，或新事物。您可以将具有时效性的 banners 和 stickers 按顺序自动插入到生成的网站。
+
+支持两种类型的 promotion:
```
Comment (gauliang) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
支持两种类型的推广:
```

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 4]
```diff
@@ -612,6 +612,64 @@ which looks like:
...
+
+- **Countdown stickers** 显示发生大事情之前还剩下多少时间。(例如，"距离 Istio 1.5发布还有2天")。在活动期间，stickers 向用户显示的视觉效果最小。
```
Comment (gauliang) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
- **Countdown stickers** 显示距离重大事件发生还有多长时间(如：“离 Istio 1.5发布还有2天”)，stickers 视觉效果对用户影响最小。
```

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 5]
```diff
@@ -612,6 +612,64 @@ which looks like:
...
+
+- **Banners**  向用户显示突出的消息，让他们知道即将发生的重大事件，正在发生或已经发生。 (例如，"Istio 1.5已发布，请立即下载！" 或 "3月30日加入我们的KubeCon")。标语是事件期间向用户显示的全屏 Banner。
```
Comment (gauliang) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
- **Banners**  视觉效果最为突出，用于向用户传达即将（已经或正在）发生的重大事件 (例如，“Istio 1.5 已发布，请立即下载！”或“3 月 30 日加入我们的 KubeCon”），其呈现效果为全屏显示 。
```

Comment (kylesliu) [reply content/zh/about/contribute/creating-and-editing-pages/index.md]: 已经修改，请review

**Pull request merged**